neqo-crypto/Cargo.toml

```diff
@@ -17,6 +17,7 @@ workspace = true
 
 [dependencies]
 enum-map = { workspace = true }
+flate2 = "1.1.1"
```

Comment:
Could we add `default-features = false` and only enable those features actually needed?